SQLite Threading Problem

I ran the the PyCA UI with `./start.sh ui` on Arch Linux and visit the website. After some reloads I got the following tracebacks multiple times:

```python
[sqlalchemy.pool.impl.NullPool:272:_close_connection()] [ERROR] Exception closing connection <sqlite3.Connection object at 0x7feb1abca6c0>
Traceback (most recent call last):
  File "/home/sven/devel/elanev/pyCA/venv/lib/python3.8/site-packages/sqlalchemy/event/base.py", line 83, in __init__
    self._empty_listeners = self._empty_listener_reg[instance_cls]
  File "/usr/lib/python3.8/weakref.py", line 383, in __getitem__
    return self.data[ref(key)]
KeyError: <weakref at 0x7feb1aaee310; to 'type' at 0x7feb14050460 (Session)>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/sven/devel/elanev/pyCA/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 697, in _finalize_fairy
    fairy._reset(pool)
  File "/home/sven/devel/elanev/pyCA/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 893, in _reset
    pool._dialect.do_rollback(self)
  File "/home/sven/devel/elanev/pyCA/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 540, in do_rollback
    dbapi_connection.rollback()
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140647752754944 and this is thread id 140647741564672.
```

There are no custom configs or an existing database before. My Environment looks like this:

```bash
$ pip freeze
click==7.1.2
configobj==5.0.6
Flask==1.1.2
itsdangerous==1.1.0
Jinja2==2.11.2
MarkupSafe==1.1.1
pycurl==7.43.0.5
python-dateutil==2.8.1
sdnotify==0.3.2
six==1.15.0
SQLAlchemy==1.3.17
Werkzeug==1.0.1
$ pacman -Q sqlite
sqlite 3.31.1-1
```

--- a/pyca/__main__.py
+++ b/pyca/__main__.py
@@ -112,7 +112,7 @@ def main():
         agentstate.run()
     elif cmd == 'ui':
         signal.signal(signal.SIGINT, signal.default_int_handler)
-        ui.app.run()
+        ui.app.run(threaded=False)
     else:
         # Invalid command
         usage(3)

